pkg/wardrobe/transfers.go

```diff
@@ -15,14 +15,18 @@ type Transfer struct {
 	Date          time.Time       `json:"date"`
 }
 
-func AddTransfer(uid string, portId int, amount decimal.Decimal, isDeposit bool, manuallyAdded bool, date time.Time) error {
+// Upserts transfer into db - function is idempotent
+func UpsertTransfer(uid string, portId int, amount decimal.Decimal, isDeposit bool, manuallyAdded bool, date time.Time) error {
```

Comment:
rekt by rebase again?

Reply:
i think it's fine? we're 0 commits behind master so...